scripts/common.sh

```diff
@@ -3,7 +3,8 @@ language=${1:-""}
 api_url=${2:-${api_url:-"https://api.cloudsmith.io/"}}
 api_version=$(curl -s "${api_url}status/check/basic/" | jq -r '.version')
 openapi_url="${api_url}?format=openapi"
-package_version="2.0.15"
+package_version="2.0.16"
+swagger_codegen_cli_image="swaggerapi/swagger-codegen-cli:2.4.43"
```

Comment:
A new variable for the swagger-codegen-cli image we wish to use. This is used in the main build.sh script, and in the build.sh scripts nested inside the $lang directories.